curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-strings-using-template-literals.md

```diff
@@ -69,14 +69,13 @@ assert(
 Template strings and expression interpolation should be used.
 
 ```js
-(getUserInput) => assert(getUserInput('index').match(/(`.*\${.*}.*`)/));
+code.match(/(`.*\${.*}.*`)/);
```

Comment:
```suggestion
assert.match(code, /(`.*\${.*}.*`)/);
```

Don't believe this was actually testing anything?

Reply:
Nope. Thanks, Naomi!